cmd/apprepository-controller-new/main.go

```diff
@@ -36,10 +48,16 @@ func init() {
 }
 
 func main() {
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	var (
+		metricsAddr          string
+		eventsAddr           string
+		enableLeaderElection bool
```

Comment:
Just out of curiosity: is this for enabling this controller to be scaled up/down ? If so, that's great. Such an improvement from our first CRD controller 😅